test/integration/breadcrumbs.cc

```diff
@@ -547,6 +550,123 @@ TEST_F(BreadcrumbsTest, AllowRenaming)
   EXPECT_TRUE(this->server->HasEntity("B1_0_1"));
 }
 
+/////////////////////////////////////////////////
+/// Return a list of model entities whose names match the given regex
```

Comment:
This could be eventually useful to move to `EntityComponentManager`. The trick is that it's probably too slow, and users may not realize that.

Reply:
Yeah, we can probably make it generic enough that it would work with any entity that has a name component.